readme - defintion of done (#24)
adaapp/Test-assignment2-team6: README.md

# Assignment2-Team6 - Currency Converter

google meet- https://meet.google.com/hjn-pqbs-fio
## Website objectives
The objective of this website is to create a currency convert that allows the user to enter a number in the textbox and it should output the conversion. The user will have a drop down box that allows them to choose between the different currencies 
## Approach to testing
the initial smoke test was completed to test if the framework we used for testing was working correctly, this was just an expect where 1 is to be 1.

## Coding standards

Variable naming - use of camel casing throughout the js

> Simple code - aiming to write simple code so that it is understandable if the website was to be maintained

Reviewer: ```suggestion
Simple code - aiming to write simple code for easier website testing and maintenance 
```
